lib/order.rb

```diff
@@ -0,0 +1,77 @@
+require_relative 'menu'
+require_relative 'text_message'
+
+class Order
+
+  attr_reader :basket, :menu
+
+  def initialize(menu = Menu.new)
+    @basket = Hash.new(0)
+    @menu = menu
+  end
+
+  def add_dish(item)
+    @menu.check_availability(item)
+    @basket[item] += @menu.dishes[item]
+  end
+
+  def remove_dish(item)
```

Comment:
Same as above so you can quickly remove the 9 items we accidentally added to the basket with the previous method :)

Reply:
True! I didn't think about that scenario!!! :D